api/pkg/apis/v1alpha1/managers/solution/solution-manager.go

```diff
@@ -456,7 +456,7 @@ func (s *SolutionManager) Reconcile(ctx context.Context, deployment model.Deploy
 		}
 		log.DebugfCtx(ctx, " M (Solution): applying step with Role %s on target %s", step.Role, step.Target)
 		someStepsRan = true
-		retryCount := 1
+		retryCount := 5
```

Comment:
@RemindD is working on retry on pubsub level, do we need to add inner retry in solution-reconcile as well?

Reply:
the status of instance will go to fail, if the retryCount is 1, the scenario test is not passed.